fuse/macfuse/Portfile

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        osxfuse osxfuse 4.1.0 macfuse-
+name                macfuse
+conflicts           osxfuse
+categories          fuse
+platforms           darwin
+supported_archs     x86_64
+license             Restrictive
+maintainers         {outlook.com:mohd.akram @mohd-akram} openmaintainer
+
+description         FUSE for macOS
+
+long_description    FUSE extends macOS by adding support for user space file \
+                    systems.
+
+homepage            https://osxfuse.github.io/
+
+github.tarball_from releases
+distname            ${name}-${version}
+use_dmg             yes
+
+checksums           rmd160  2c5e9741b983e660e34139966838af8a49eb3b5c \
+                    sha256  3cb6a49406fd036c50ef1b4ad717a377f4dcf182811bde172d69f1c289791085 \
+                    size    5747012
+
+if {${os.platform} eq "darwin" && ${os.major} < 16} {
+    known_fail yes
+    pre-fetch {
+        ui_error "${name} is only supported on macOS 10.12 or later."
```

Comment:
Does the osxfuse port even work for older versions of macOS? There seems to be a bunch of issues on the tracker.

Reply:
I used it on macOS 10.14 and seems to work. Anyway, the new macfuse which version support? and the older one (osxfuse)? because if they are superimposable we can substitute the older osxfuse